.github/workflows/format.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   build:
     name: Code is formatted
-    runs-on: ubuntu-latest
```

Comment:
Although this is still a draft PR, why is this being changed? I prefer sticking to `ubuntu-latest` because at least that way we can pick up any potential issues and isolate them

Reply:
I tried to keep it in sync across the project. I usually prefer fixed versions to have builds consistent. It is much harder to track sudden build failure if the change is not obvious and deliberate.